src/generate-changelog.ts

```diff
@@ -185,80 +185,51 @@ async function getReleases(graphqlWithAuth: Function): Promise<ReleaseInfo[]> {
+  if (matches) {
+    const matchesArray = [...matches]
+    for (const match of matchesArray) {
+      const matchString = match[1].trim()
+      if (
+        matchString &&
+        !CHANGELOG_TEMPLATE_TEXT.test(matchString)
+      ) {
+        changelogSectionArray.push(matchString)
+      }
+    }
+  }
+}
+
 // Produces the changelog from an array of PR descriptions. At a high level, we
-// traverse each PR description searching for `CHANGELOG-FIXES:` or `CHANGELOG-ADDS:`
-// to determine what the changelog conntents should be.
+// traverse each PR description searching for different changelog prefix strings
+// to determine what the changelog contents should be.
 function parseChangelogFromPrDescriptions(prDescriptions: string[]): Changelog {
-  const changelog_fixed: string[] = []
-  const changelog_new: string[] = []
-  const teams_specific_changes: string[] = [];
-  const changelog_image: string[] = [];
+  const changelogNewFeatures: string[] = []
+  const changelogImprovements: string[] = []
+  const changelogBugFixes: string[] = [];
+  const changelogImages: string[] = [];
 
   for (const prDescription of prDescriptions) {
-    const fixMatches = prDescription.matchAll(FIXES_REGEX)
-    if (fixMatches) {
-      const fixMatchesArray = [...fixMatches]
-      for (const fixMatch of fixMatchesArray) {
-        const fixMatchString = fixMatch[1].trim()
-        if (
-          fixMatchString &&
-          !CHANGELOG_TEMPLATE_TEXT.test(fixMatchString)
-        ) {
-          changelog_fixed.push(fixMatchString)
-        }
-      }
-    }
-
-    const addMatches = prDescription.matchAll(NEW_REGEX)
-    if (addMatches) {
-      const addMatchesArray = [...addMatches]
-      for (const addMatch of addMatchesArray) {
-        const addMatchString = addMatch[1].trim()
-        if (
-          addMatchString &&
-          !CHANGELOG_TEMPLATE_TEXT.test(addMatchString)
-        ) {
-          changelog_new.push(addMatchString)
-        }
-      }
-    }
-    
-    const teamsMatches = prDescription.matchAll(TEAMS_SPECIFIC_REGEX);
-        if (teamsMatches) {
-            const teamsMatchesArray = [...teamsMatches];
-            for (const teamsMatch of teamsMatchesArray) {
-                const teamsMatchString = teamsMatch[1].trim();
-                if (teamsMatchString &&
-                    !CHANGELOG_TEMPLATE_TEXT.test(teamsMatchString)) {
-                      teams_specific_changes.push(teamsMatchString);
-                }
-            }
-        }
-    
-    const imageMatches = prDescription.matchAll(IMAGE_REGEX);
-        if (imageMatches) {
-          const imageMatchesArray = [...imageMatches];
-          for (const imageMatch of imageMatchesArray) {
-            const imageMatchString = imageMatch[1].trim();
-            if (imageMatchString &&
-              !CHANGELOG_TEMPLATE_TEXT.test(imageMatchString)) {
-                changelog_image.push(imageMatchString);
-            }
-          }
```

Comment:
yikes so many of these were just copy-pasted, thanks for adding a helper function